pkg/services/temp_user/tempuserimpl/store_test.go

```diff
@@ -56,6 +56,29 @@ func TestIntegrationTempUserCommandsAndQueries(t *testing.T) {
 		require.Nil(t, err)
 		require.Equal(t, 1, len(queryResult))
 	})
+	t.Run("Should not be able to get temp users by case-insentive email - case sensitive", func(t *testing.T) {
+		if db.IsTestDbMySQL() {
+			t.Skip("MySQL is case insensitive by default")
+		}
+		setup(t)
+		store.cfg.CaseInsensitiveLogin = false
+		query := tempuser.GetTempUsersQuery{Email: "E@as.co", Status: tempuser.TmpUserInvitePending}
+		queryResult, err := store.GetTempUsersQuery(context.Background(), &query)
+
+		require.Nil(t, err)
+		require.Equal(t, 0, len(queryResult))
+	})
+
+	t.Run("Should be able to get temp users by email - case insensitive", func(t *testing.T) {
+		setup(t)
+		store.cfg.CaseInsensitiveLogin = true
+		query := tempuser.GetTempUsersQuery{Email: "E@as.co", Status: tempuser.TmpUserInvitePending}
+		queryResult, err := store.GetTempUsersQuery(context.Background(), &query)
+
+		require.Nil(t, err)
+		require.Equal(t, 1, len(queryResult))
+		store.cfg.CaseInsensitiveLogin = false
```

Comment:
nit; better reset to `false` inside `t.Cleanup()`

Reply:
@papagian you mean like this?
```
		t.Cleanup(func() {
			store.cfg.CaseInsensitiveLogin = false
		})
```

Reply:
yes!